src/ts/scene/CharacterScene.ts

```diff
@@ -39,62 +39,46 @@ export default class CharacterScene extends Scene
     
     create(): void
     {
-        this.add.image(300, 400, 'SBGI').setDepth(0);
-        
-        var sprite1 = this.add.sprite(220, 600, 'smugglerSel').setInteractive().setDepth(2);
-        sprite1.on('pointerover', function (this: Phaser.GameObjects.Sprite){
-            this.setTint(0x95dccf);
-        });
-        sprite1.on('pointerout', function (this: Phaser.GameObjects.Sprite){
-            this.clearTint();
-        });
-        sprite1.on('pointerup', function (this: Phaser.GameObjects.Sprite){
-            this.scene.add.image(0,0,'smugglerBig').setDepth(1).setOrigin(0);
-            
-        });
-
+    
+        this.add.image(300, 400, 'SBGI').setDepth(1).setVisible(true)
 
-        var sprite2 = this.add.image(420, 600, 'phantomSel').setInteractive().setDepth(2);
-        sprite2.on('pointerover', function (this: Phaser.GameObjects.Sprite){
-            this.setTint(0x95dccf);
-        });
-        sprite2.on('pointerout', function (this: Phaser.GameObjects.Sprite){
-            this.clearTint();
-        });
-        sprite2.on('pointerup', function (this: Phaser.GameObjects.Sprite){
-            this.scene.add.image(0,0,'phantom').setDepth(1).setOrigin(0);
-        });
+        const button1 = this.add.image(220, 600, 'smugglerSel').setInteractive().setDepth(3);
+        const button2 = this.add.image(420, 600, 'phantomSel').setInteractive().setDepth(3);
+        const button3 = this.add.image(620, 600, 'WarwickSel').setInteractive().setDepth(3);
```

Comment:
반복되는 코드는 리스트로 묶어서 처리하는 것이 좋습니다.

```suggestion
        const buttons = [
                this.add.image(420, 600, 'phantomSel'),
                this.add.image(420,600, 'phantomSel'),
                this.add.image(620, 600, 'WarwickSel'),
        ].map(button => button.setInteractive().setDepth(3));
```